CMakeLists.txt

```diff
@@ -376,3 +377,32 @@ add_custom_target(
   COMMAND ${CMAKE_COMMAND} -DCOMPONENT=viam-cpp-sdk_examples -P ${CMAKE_BINARY_DIR}/cmake_install.cmake
   DEPENDS all
 )
+
+add_custom_target(
+  install-tests
```

Comment:
There is a decision to be made here. Should tests be installed / installable? If yes, should they be run from the installation directory or from the build directory?

Overall, I'm not sure it is particularly useful to install the tests. And I think it is harder to make the test execution targets make use of the installed tests, and easier to make it use the test binaries out of the build tree (I think that is what is happening now).